os/pm/pm_changestate.c

```diff
@@ -227,6 +233,42 @@ int pm_changestate(enum pm_state_e newstate)
 	 * drivers may refuse the state change.
 	 */
 	if (newstate != PM_RESTORE) {
+
+		/* If new state is same as current state then do nothing*/
+		if (newstate == g_pmglobals.state) {
+			ret = OK;
+			goto EXIT;
+		}
+
+		/* If LCD invokes the state change then change the LCD Power */
+		else if (newstate == PM_NORMAL || newstate == PM_IDLE || (g_pmglobals.state == PM_IDLE && newstate == PM_STANDBY)) {
```

Comment:
I have been doing this for LCD + PM. Please find here: https://github.com/Samsung/TizenRT/pull/6221/files#diff-1ac6711480a69e526c929a7ee70f7a696b05cf8eff0dd0e6e91bb5f260776007R292
Besides, could you please let me know where is the decision maker for LCD behaviour?

Hi @sunghan-chang @ewoodev ,
I also used fd open/closed in driver/kernel, for some operations which is not directly callable. Could you let me know whether is it feasible to do so?

Reply:
Thank you for raise the PR about the LCD + PM.
I'm looking your PR, Let me leave the comment, we can discuss it in your PR.